src/libstore/ssh-store.cc

```diff
@@ -17,6 +17,7 @@ struct SSHStoreConfig : virtual RemoteStoreConfig
     const Setting<bool> compress{(StoreConfig*) this, false, "compress", "whether to compress the connection"};
     const Setting<Path> remoteProgram{(StoreConfig*) this, "nix-daemon", "remote-program", "path to the nix-daemon executable on the remote system"};
     const Setting<std::string> remoteStore{(StoreConfig*) this, "", "remote-store", "URI of the store on the remote system"};
+    const Setting<int> logFD{(StoreConfig*) this, -1, "log-fd", "file descriptor to which SSH's stderr is connected"};
```

Comment:
Regardless of whether it's a good idea to set log-fd to 4 for ssh-ng, it shouldn't be a configurable option.